manifests/init.pp

```diff
@@ -42,7 +42,7 @@
   $policies_yaml = inline_template('<%= Hash[policies_array.map { |p|
       ["foreman_scap_client_#{p["id"]}",
         {
-          "command" => "/usr/bin/foreman_scap_client #{p[\'id\']}",
+          "command" => "/usr/bin/foreman_scap_client #{p[\'id\']} > /dev/null",
```

Comment:
you don't want to send mail notifications? if so, maybe you should also redirect 2> /dev/null